aztec/src/main/kotlin/org/wordpress/aztec/spans/AztecBackgroundColorSpan.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.aztec.spans
+
+import android.graphics.Color
+import android.text.TextPaint
+import android.text.style.BackgroundColorSpan
+import org.wordpress.aztec.AztecAttributes
+
+class AztecBackgroundColorSpan(
+        val color: Int,
+        val alpha: Int = 220,
```

Comment:
I think only `color` is ever used as a constructor parameter from here. The other parameters can probably be moved to be class properties. What do you think?

Reply:
the color is used for the super class, BackgroundColorSpan(color) constructor

Reply:
no problem, i'll take a look into this